frontend/src/helpers/helpers.ts

```diff
@@ -212,3 +212,19 @@ export function searchAnatomicalEntities(
     )
     .sort((a, b) => a.name.localeCompare(b.name));
 }
+
+
+export function getForwardConnectionText(connections: any[]) {
+  const BASE_FORWARD_CONNECTION_TEXT = "This neuron population connects to connectivity statement with id:";
+  let text = ''
+  if (connections.length) {
+    text += BASE_FORWARD_CONNECTION_TEXT
+    connections.slice(0, connections.length - 1).forEach((connection: any) => {
+      text = text.concat(` <a style="text-decoration: none" href="/statement/${connection.id}">${connection.id}</a>,`);
+    });
+    text = text.slice(0, -1);  // remove the last comma
```

Comment:
question: @D-GopalKrishna consider the length of the list is 1. Do you think this line and the lines below are correct?

Reply:
thanks a lot, @zsinnema! It didn't address the case for 1 connection yes. I corrected it along with Dario's feedback below. Thanks again.